sonic_platform_base/sfp_base.py

```diff
@@ -0,0 +1,204 @@
+
+        Returns:
+            A Boolean, True if SFP has rx los, False if not.
+            Note : rx los status is lached until a call to get_rx_los or a reset.
+        """
+        return NotImplementedError
+
+    def get_tx_fault(self):
+        """
+        Retrieves the tx fault status of SFP
+
+        Returns:
+            A Boolean, True if SFP has tx-fault, False if not
+            Note : tx fault status is lached until a call to get_tx_fault or a reset.
+        """
+        return NotImplementedError
+
+    def get_tx_disable(self):
+        """
+        Retrieves the tx-disable status of this SFP
+
+        Returns:
+            A Boolean, True if tx-disable is enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def get_tx_disable_channel(self):
+        """
+        Retrieves the tx disabled channels in this SFP
+
+        Returns:
+            A hex of 4 bits (bit 0 to bit 3 as channel 0 to channel 3) to represent
+            tx channels which have been disabled in this SFP.
+            As an example, a returned value of 0x5 indicates that channel 0 
+            and channel 2 have been disabled.
+        """
+        return NotImplementedError
+
+    def get_lpmode(self):
+        """
+        Retrieves the lpmode status of this SFP
+
+        Returns:
+            A Boolean, True if lpmode (low power mode) is enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def get_power_override(self):
+        """
+        Retrieves the power-override status of this SFP
+
+        Returns:
+            A Boolean, True if power-override is enabled, False if disabled
+        """
+        return NotImplementedError
+
+    def reset(self):
+        """
+        Reset SFP and return all user module settings to their default srate.
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        return NotImplementedError
+    
+    def tx_disable(self, tx_disable):
+        """
+        Disable SFP tx for all channels
+
+        Args:
+            tx_disable : A Boolean, True to enable tx_disable mode, False to disable
+                         tx_disable mode.
+
+        Returns:
+            A boolean, True if tx_disable is set successfully, False if not
+        """
+        return NotImplementedError
+    
+    def tx_disable_channel(self, channel, disable):
+        """
+        Sets the tx_disable for specified SFP channels
+
+        Args:
+            channel : A hex of 4 bits (bit 0 to bit 3) which represent channel 0 to 3,
+                      e.g. 0x5 for channel 0 and channel 2.
+            disable : A boolean, Trure to disable tx channels specified in channel,
+                      False to enable
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        return NotImplementedError
+    
+    def set_lpmode(self, lpmode):
+        """
+        Sets the lpmode of SFP
```

Comment:
As with my comment on `get_lpmode()` above, move the parenthetical full description `(low power mode)` from the Args comment on line 180 below to this line, as this line comes first.

Reply:
Moved full descriptions to proper location.